src/video_core/amdgpu/liverpool.h

```diff
@@ -1088,6 +1088,15 @@ struct Liverpool {
         submit_cv.notify_one();
     }
 
+    void reserveCopyBufferSpace() {
+        GpuQueue& gfx_queue = mapped_queues[GfxQueueId];
+        std::scoped_lock<std::mutex> lk(gfx_queue.m_access);
+
+        constexpr size_t gfx_reserved_size = (2 << 21) / sizeof(u32); // 2MB
```

Comment:
I would prefer simple `2_MB` here.  Also, we're using camel case for constants naming